src/components/videoCard/videoCard.css

```diff
@@ -81,7 +82,9 @@
     background-color: white;
     padding: 3px 0;
     position: absolute;
-    left: 9rem;
+    left: 8rem;
+    top: 11rem;
+    max-width: 10rem;

```

Comment:
"Remove from watch later" in the menu options of the card is going into the next line. You can remove the `max-width` and decrease the `left` property.

Reply:
Sure, will do.